src/info.c

```diff
@@ -454,6 +454,11 @@ void info_init(const struct config* cfg)
         config_get_num(cfg, CFG_INFO, CFG_INFO_STIMEOUT, 0, 1024);
     timeout_init(&ctx.status);
 
+    info_on_scale();
+}
+
+void info_on_scale(void)
```

Comment:
The key size has been updated, but all values ​​remain the same.

Reply:
Do you mean the values that come after the keys? They'll be redrawn on the following `app_redraw()` call.